helpers/mysql_stored_procedures.go

```diff
@@ -0,0 +1,148 @@
+package helpers
```

Comment:
Maybe we can put the stored procedures into dedicated files (where the filename equals the procedure name). This would allow for better IDE support, I guess.

It would then be good to also have the Postgres procedures in dedicated files to easily compare them.

Reply:
Yes, thought about that. The order of the files shouldn't matter, so we could just place them in a folder and read them all.